packages/web-shared/components/ContentSingle.js

```diff
@@ -5,6 +5,7 @@ import { Helmet } from 'react-helmet';
 
 import { getURLFromType } from '../utils';
 
+import moment from 'moment';
```

Comment:
@rajitharussel is a way we could use `date-fns` instead of `moment`? 

Moment is pretty big package and I've been meaning to remove it completely from the codebase.

Reply:
done